### PR TITLE
Bug 2017016: Moving virtualization to a new menu section

### PR DESCRIPTION
**Pull request by metalice**
Signed-off-by: Matan Schatzman <mschatzm@redhat.com>

image

image

### CODE REVIEW

**Comment by rawagner** [comment]
Since we are just beginning 4.10 I do not like that you're putting workaround to code for nav item ordering.
It would be best to add ordering of nav items feature first.

File: frontend/public/components/nav/admin-nav.tsx [hunk 1]
```diff
@@ -169,6 +166,12 @@ const AdminNav = () => {
 
       {/* Temporary addition of Knative Serverless section until extensibility allows for section ordering
           and admin-nav gets contributed through extensions. */}
+      <NavSection
```
[the remark this answers is not in the document]
Comment (metalice) [reply frontend/public/components/nav/admin-nav.tsx:179]: you right! i added to the comment Virtualization as it related to us also :)

**Pull request merged**